sql/server.go

```diff
@@ -570,7 +570,7 @@ func (s *Server) Set(session *Session, p *parser.Set, args []driver.Datum,
 	// be double quoted.
 	name := strings.ToLower(p.Name.String())
 	switch name {
-	case `"database"`: // Quoted: database is a reserved word
+	case `"database"`, `database`:
```

Comment:
I don't think this is the right place to do this. @petermattis care to weigh in?